### ISSUE TITLE
Quickstart documentation steps are incorrect

### ISSUE THREAD

**Issue by ryanulit** [opentdf/platform#1269]
The current [Quickstart docs](https://github.com/opentdf/platform?tab=readme-ov-file#quick-start) have a few issues:

If you attempt to run the `docker compose` step without first generating dev keys/certs, you will get the following error:

![image](https://github.com/user-attachments/assets/36bb7028-b9e1-4b76-bf01-37533d35a14e)

This also leaves your `./keys` directory in a bad state (see below), and running the `./.github/scripts/init-temp-keys.sh` script won't fix this because any existing files don't seem to get recreated.  You have to delete the directory and rerun.

![image](https://github.com/user-attachments/assets/f076101c-97ab-411e-bf90-e4c917b51dbf)

The other issue is that the `opentdf-example.yaml` config file uses a `keycloak` host in numerous places that isn't accessible by the go service when running `go run ./service start`.  You need to use `localhost`, which is fixed by copying the `opentdf-dev.yaml` file into your local `opentdf.yaml`.

In conclusion, the proper steps should be:
1. create dev keys/certs before anything else
2. docker compose up
3. copy `opentdf-dev.yaml` into local `opentdf.yaml`
4. run the keycloak provision command
5. start the platform service
